.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+on: push
+jobs:
+  test:
+    strategy:
+      matrix:
+        go: [ '1.10.x', '1.11.x', '1.12.x' ]
+        platform: [ubuntu-latest, macos-latest]
```

Comment:
You can increase of Windows tests, but the tests will fail.

Reply:
What’re causes of failure?

Reply:
This is a failed reason.
https://github.com/swaggo/swag/pull/528/checks?check_run_id=232011601